r/R/dplyr.R

```diff
@@ -264,17 +264,24 @@ arrow_eval <- function (expr, mask) {
     # else, for things not supported by Arrow return a "try-error",
     # which we'll handle differently
     msg <- conditionMessage(e)
-    # TODO(ARROW-11700): internationalization
-    if (grepl("object '.*'.not.found", msg)) {
-      stop(e)
-    }
-    if (grepl('could not find function ".*"', msg)) {
+    patterns <- paste(i18ize_error_messages(), collapse = "|")
```

Comment:
I want to move this to something that is evaluated once instead of every time we evaluate an expression.